db/osqlsession.h

```diff
@@ -141,27 +83,6 @@ int osql_sess_getcrtinfo(void *obj, void *arg);
  */
 void *osql_sess_getbptran(osql_sess_t *sess);
 
-/* Lock the session */
```

Comment:
all these were a sign of bad encapsulation;  these are gone, since knowledge of the locking and the implied fields is isolated to osqlsession.c